it/mail/chrome/messenger/tabmail.dtd

```diff
@@ -1,12 +1,12 @@
-<!ENTITY  untitledTab              "(Untitled)">
+<!ENTITY  untitledTab              "(senza titolo)">
```

Comment:
CORRECTION NEEDED. 
The English version uses a capital letter (Untitled). I would do the same in Italian.